### PR TITLE
[Fix] Shared object name change

### PR DESCRIPTION
**Pull request by stealthcopter**
Whoops, when we moved to cmake the .so library name changed, so it broke. This fix addresses that

### CODE REVIEW

File: app/src/main/res/menu/menu_main.xml [hunk 1]
```diff
@@ -9,6 +9,13 @@
...
     app:showAsAction="always" />
 
+  <item
+      android:id="@+id/action_export_logs"
+      android:icon="@drawable/ic_logs_24"
+      android:orderInCategory="100"
+      android:title="@string/action_info"
+      app:showAsAction="ifRoom" />
+
   <item
```
Comment (stealthcopter) [inline app/src/main/res/menu/menu_main.xml:19]:
```suggestion
  <item
```

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -4,6 +4,7 @@
     <string name="action_settings">Settings</string>
     <string name="action_github">Get on GitHub</string>
     <string name="action_info">info</string>
+    <string name="action_export_logs">View Logs</string>
```
Comment (stealthcopter) [inline app/src/main/res/values/strings.xml:7]:
```suggestion
```

**Review by stealthcopter — COMMENTED** [review]
removed files that weren't supposed to be in this commit

File: app/src/main/res/menu/menu_main.xml [hunk 3]
```diff
@@ -9,6 +9,13 @@
     android:title="@string/action_info"
     app:showAsAction="always" />
 
+  <item
+      android:id="@+id/action_export_logs"
+      android:icon="@drawable/ic_logs_24"
+      android:orderInCategory="100"
+      android:title="@string/action_info"
+      app:showAsAction="ifRoom" />
+
```
Comment (stealthcopter) [inline app/src/main/res/menu/menu_main.xml:18]:
```suggestion
```

**Review by scottyab — APPROVED** [review]
Thanks for fixing this @stealthcopter I was was failing to multi task the other day.

**Pull request merged**